.github/workflows/static_code_checks.yaml

```diff
@@ -48,3 +48,4 @@ jobs:
             GHSA-3ww4-gg4f-jr7f
             GHSA-9v9h-cgj8-h64p
             GHSA-6vqw-3v5j-54x4
+            GHSA-7gpw-8wmc-pm8g
```

Comment:
Looks like we can just upgrade `aiohttp` to latest and remove this from the ignore list.

```
poetry update aiohttp
```

https://github.com/advisories/GHSA-7gpw-8wmc-pm8g

Reply:
Yeah will do that before merging!